webapp/src/components/conference/conference.tsx

```diff
@@ -128,6 +135,7 @@ export default class Conference extends React.PureComponent<Props, State> {
     }
 
     componentDidMount() {
+        document.addEventListener('keydown', this.escFunction, false);
```

Comment:
Shouldn't we also remove the listener on componentDidUnmount?

Reply:
done